src/Tizen.System/PowerUsage/PowerUsage.cs

```diff
@@ -0,0 +1,224 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace Tizen.System
+{
+    static internal class Globals
+    {
+        internal const string LogTag = "Tizen.System.PowerUsage";
```

Comment:
This means `Tizen.System.Globals.LogTag = "Tizen.System.PowerUsage"`
Is it correct? I think it can be moved to inside of PowerUsages class

Reply:
yes. It can be moved.
Will move that.